sample-apps/spark/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
```

Comment:
Ancient gradle version! Let's not have subgradle wrappers for the sample apps since they're not necessarily for user consumption, e.g., copy-paste and use. So just make them subprojects of the top project.

Reply:
good catch!